lib/Nagios/Check/RabbitMQ/Types.hs

```diff
@@ -38,14 +34,25 @@ data CheckOptions = CheckOptions
     , maxCritical :: Threshold
     } deriving Show
 
+data ConnectionDetail = ConnectionDetail
+    { publish :: Double } deriving (Show, Generic)
+
+instance FromJSON ConnectionDetail where
+    parseJSON (Object o) = ConnectionDetail
+         <$> ((o .: "stats") >>= (.: "publish"))
+
 data MessageDetail = MessageDetail
-    { rateConfirms   :: Double
-    , ratePublishIn  :: Double
-    , ratePublishOut :: Double
+    { rateConfirms        :: Double
+    , ratePublishIn       :: Double
+    , ratePublishOut      :: Double
+    , connectionsIncoming :: [ConnectionDetail]
+    , connectionsOutgoing :: [ConnectionDetail]
     } deriving (Show,Generic)
 
 instance FromJSON MessageDetail where
     parseJSON (Object o) = MessageDetail
-                        <$> ((o .: "message_stats") >>= (.: "confirm_details") >>= (.: "avg_rate"))
-                        <*> ((o .: "message_stats") >>= (.: "publish_in_details") >>= (.: "avg_rate"))
-                        <*> ((o .: "message_stats") >>= (.: "publish_out_details") >>= (.: "avg_rate"))
+	<$> ((o .: "message_stats") >>= (.: "confirm_details") >>= (.: "avg_rate"))
```

Comment:
Do you know what time period these `avg_rate`s are over? Might be good to link to the appropriate doc section here or something.